lib/features/children/family_history/family_history.dart

```diff
@@ -78,6 +79,26 @@ class FamilyHistory extends StatelessWidget {
                       return getDivider(state, index);
                     },
                   ),
+                  if (state.history.isEmpty)
```

Comment:
I would suggest to extract it to a separate HistoryEmptyStateWidget (name is just en example)